src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java

```diff
@@ -74,10 +74,10 @@
     private static final String JENKINS_HOME = "JENKINS_HOME=";
     private static final Logger LOGGER = Logger.getLogger(ContainerExecDecorator.class.getName());
 
-    private final transient KubernetesClient client;
+    private transient KubernetesClient client;
 
     @SuppressFBWarnings(value = "SE_TRANSIENT_FIELD_NOT_RESTORED", justification = "not needed on deserialization")
-    private final transient List<Closeable> closables = new ArrayList<>();
```

Comment:
Why?

Reply:
When the pipeline is resumed, since it is transient, it will not be deserialized...therefore when we come to add to array, it will NPE...therefore, we check before we add to it and create if necessary. Therefore it cannot be final.

Reply:
Makes sense to me.